### DISCUSSION TITLE
AI Deck stuck at 99% when flashing the wifi streamer example

### DISCUSSION THREAD

**Discussion by Kaijun101** [bitcraze/discussions#907] [Q&A]
Hi,

I have tried to flash the wifi streamer example as indicated https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/simple-examples/wifi-streamer/. However, when it flashes it is relatively slow. 

So, we have tried to restore the firmware through DFU mode with no decks attached then update the STM32 with the AI-deck through CFClient. Neither do we have a JTag. Therefore, can you advise how to resolve this problem without the JTag connector.

The AI-decks are newly bought.

**Comment by krichardsson** [comment]
Duplicate of #721 and #574
